eng/templates/Azure.ResourceManager.Template/tests/Azure.ResourceManager.Template.Tests.csproj

```diff
@@ -1,12 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TestHelperProjects>;</TestHelperProjects>
+    <UseNewMgmtFramework>true</UseNewMgmtFramework>
```

Comment:
this shouldn't be needed we have a task to flip this so its default to include if mgmtlibrary